--- a/custom_logic/service_report_hooks.py
+++ b/custom_logic/service_report_hooks.py
@@ -55,7 +55,7 @@
 
     req_status = frappe.db.get_value("Service Request", doc.service_request, "status")
 
-    if not req_status:
+    if req_status is None:
         frappe.logger(__name__).error(
             _(
                 "Не удалось получить статус для заявки '{0}', связанной с отчетом '{1}'."
@@ -124,7 +124,7 @@
     except Exception as e:
         frappe.logger(__name__).error(
             _("Ошибка при обновлении заявки '{0}' из отчета '{1}': {2}").format(
-                doc.service_request, doc.name, e
+                doc.service_request, doc.name, str(e)
             ),
             exc_info=True,
         )
@@ -132,4 +132,4 @@
             _(
                 "Произошла ошибка при обновлении связанной заявки. Обратитесь к администратору."
             )
-        )
+        )