--- a/ferum_customs/doctype/service_report_document_item/service_report_document_item.py
+++ b/ferum_customs/doctype/service_report_document_item/service_report_document_item.py
@@ -1,13 +1,16 @@
 # Copyright (c) 2025 Ferum LLC and contributors
 # For license information, please see license.txt
 
-# import frappe
+import frappe
 from frappe.model.document import Document
 
 
-class ServiceReportDocumentItem(Document):  # type: ignore[misc]
-    # Эта строка (pass) означает, что в этом классе нет специфической логики.
-    # Frappe будет использовать стандартное поведение для дочерних документов.
-    # Если вам понадобится добавить валидации или другую логику для каждой строки
-    # этой таблицы на сервере, вы можете сделать это здесь.
-    pass
+class ServiceReportDocumentItem(Document):
+    # This class currently has no specific logic.
+    # Frappe will use the standard behavior for child documents.
+    # If you need to add validations or other logic for each row
+    # of this table on the server, you can do it here.
+    
+    def validate(self):
+        # Example validation logic can be added here
+        pass