--- a/ferum_customs/ferum_customs/doctype/service_report_work_item/service_report_work_item.py
+++ b/ferum_customs/ferum_customs/doctype/service_report_work_item/service_report_work_item.py
@@ -1,13 +1,11 @@
 # Copyright (c) 2025 Ferum LLC and contributors
 # For license information, please see license.txt
 
-# import frappe
 from frappe.model.document import Document
 
-
-class ServiceReportWorkItem(Document):  # type: ignore[misc]
-    # Эта строка (pass) означает, что в этом классе нет специфической логики.
-    # Frappe будет использовать стандартное поведение для дочерних документов.
-    # Если вам понадобится добавить валидации или другую логику для каждой строки
-    # этой таблицы на сервере, вы можете сделать это здесь.
-    pass
+class ServiceReportWorkItem(Document):
+    # This class currently does not contain any specific logic.
+    # Frappe will use the default behavior for child documents.
+    # If you need to add validations or other logic for each row
+    # of this table on the server, you can do so here.
+    pass

