--- a/ferum_customs/patches/create_custom_roles_and_permissions.py
+++ b/ferum_customs/patches/create_custom_roles_and_permissions.py
@@ -3,6 +3,7 @@
 from __future__ import annotations
 
 import frappe
+from typing import Union
 
 ROLES = [
     "Проектный менеджер",
@@ -10,7 +11,7 @@
     "Заказчик",
 ]
 
-SERVICE_REQUEST_PERMS: list[dict[str, int | str]] = [
+SERVICE_REQUEST_PERMS: list[dict[str, Union[int, str]]] = [
     {
         "parent": "Service Request",
         "role": "Проектный менеджер",
@@ -74,7 +75,7 @@
     role.insert(ignore_permissions=True)
 
 
-def create_docperm(perm: dict[str, int | str]) -> None:
+def create_docperm(perm: dict[str, Union[int, str]]) -> None:
     """Insert DocPerm if it is missing."""
     filters = {
         "parent": perm["parent"],
@@ -102,4 +103,4 @@
         create_role(role)
 
     for perm in SERVICE_REQUEST_PERMS:
-        create_docperm(perm)
+        create_docperm(perm)
