--- a/ferum_customs/patches/create_custom_roles_and_permissions.py
+++ b/ferum_customs/patches/create_custom_roles_and_permissions.py
@@ -68,7 +68,7 @@
 def create_role(role_name: str) -> None:
     """Ensure that a Role with the given name exists."""
 
-    if frappe.db.exists("Role", role_name):
+    if frappe.db.exists("Role", {"role_name": role_name}):
         return
 
     role = frappe.get_doc({"doctype": "Role", "role_name": role_name})
@@ -105,4 +105,4 @@
         create_role(role)
 
     for perm in SERVICE_REQUEST_PERMS:
-        create_docperm(perm)
+        create_docperm(perm)
