--- a/ferum_customs/patches/create_custom_roles_and_permissions.py
+++ b/ferum_customs/patches/create_custom_roles_and_permissions.py
@@ -67,7 +67,6 @@
 
 def create_role(role_name: str) -> None:
     """Ensure that a Role with the given name exists."""
-
     if frappe.db.exists("Role", role_name):
         return
 
@@ -77,7 +76,6 @@
 
 def create_docperm(perm: dict[str, int | str]) -> None:
     """Insert DocPerm if it is missing."""
-
     filters = {
         "parent": perm["parent"],
         "role": perm["role"],
@@ -100,9 +98,8 @@
 
 def execute() -> None:
     """Create default roles and permissions for the app."""
-
     for role in ROLES:
         create_role(role)
 
     for perm in SERVICE_REQUEST_PERMS:
-        create_docperm(perm)
+        create_docperm(perm)
