--- a/ferum_customs/ferum_customs/doctype/service_report/service_report.py
+++ b/ferum_customs/ferum_customs/doctype/service_report/service_report.py
@@ -47,7 +47,7 @@
         posting_date_val = self.get("posting_date")
         if posting_date_val:
             if not isinstance(posting_date_val, str):
-                if isinstance(posting_date_val, datetime.datetime | datetime.date):
+                if isinstance(posting_date_val, (datetime.datetime, datetime.date)):
                     self.posting_date = posting_date_val.isoformat()
         else:
             if self.is_new() and self.meta.get_field("posting_date").reqd:
@@ -146,4 +146,4 @@
         from ferum_customs import api
 
         self.calculate_totals()
-        return cast(str, api.create_invoice_from_report(self.name))
+        return cast(str, api.create_invoice_from_report(self.name))
