--- a/ferum_customs/permissions/permissions.py
+++ b/ferum_customs/permissions/permissions.py
@@ -42,6 +42,6 @@
 
     if is_customer_role:
         # Пользователь с ролью "Заказчик" и привязанным клиентом видит только заявки своего клиента.
-        return {"custom_customer": user_linked_customer}
+        return {"custom_customer": ("=", user_linked_customer)}
 
-    return None
+    return None
